### PR TITLE
Provider Organization Sync Feature Documentaion

### PR DESCRIPTION
**Pull request by gustavorodriguespereira**
- First commit 518c281319bfe6d5bdcaa83d3d9cb61f0625142e, we have the config overview
- Second commit ca2bbf80e2b8fab30047aa5b65dd9d9c7c336f8b, we have the endpoints

### CODE REVIEW

File: docs/configurations_overview.md [hunk 1]
```diff
@@ -364,6 +365,10 @@ The retroactive time period (in months) to fetch irrigation activities from prov
 
 ### Synchronization
 
+#### organizationsDataSync
+
+If set to `ALL`, Leaf will fetch organizations from the provider account with the default processing behavior for the resources with the status set as `SELECTED`. When set to `SELECTED_ONLY` the API will ingest the Organizations for each Provider which should be the top-level entity that holds the FMIS structure resources in a `PREVIEW` status. Users can then manage the sync of the data from each organization using the [Organization Sync][18] endpoints
```
Comment (awimbush) [inline docs/configurations_overview.md:370]: If set to 'ALL', Leaf will fetch and process data from all organizations within the provider account.  If set to 'SELECTED_ONLY', Leaf will only fetch and process data from the organizations were the organization status has been set to 'SELECTED'.  Setting the status of an organization is done using the Organization Sync endpoints.

File: docs/provider_organizations.md [hunk 2]
```diff
@@ -8,11 +8,21 @@ import Tabs from '@theme/Tabs';
...
+[4]: #get-all-provider-organizations
+[5]: #get-a-provider-organization
+[6]: #update-a-provider-organization-status
+[7]: https://withleaf.io/en/tutorials/john-deere-authentication-with-leaf/
+
 
 ## About
 
-The endpoints described here are tools for listing organizations that have or have not granted access to Leaf.
+The endpoints described here are separated into two disticnt categories, the [Organizations List][2] are tools for listing organizations that have or have not granted access to Leaf. Additionaly we have the [Organization Sync][3] which are tools to manage the Organizations to be processed which sequently will impact on the associated resources such as Field Boundaries, Operations and Asset for example.
```
Comment (awimbush) [inline docs/provider_organizations.md:25]: The endpoints described here are divided into two categories. The [Organizations List][2] provides tools for listing organizations that have or have not granted access to Leaf. The [Organization Sync][3] offers tools for managing organizations to be processed, which in turn impacts associated resources such as field boundaries, operations, and assets.

Comment (gustavorodriguespereira) [reply docs/provider_organizations.md:25]: Changes executed with the 46c5dfefa9674ecb474c230a595c724fd044ca36 commit

File: docs/configurations_overview.md (hunk 1 above)
Comment (gustavorodriguespereira) [reply docs/configurations_overview.md:370]: Changes executed with the 46c5dfefa9674ecb474c230a595c724fd044ca36 commit

**Review by guthierry — APPROVED** [review]

File: docs/provider_organizations.md [hunk 3]
```diff
@@ -118,3 +131,236 @@ At Trimble it is not possible to differentiate which organizations are connected
...
 
+## Organizations Sync 
+
+:::info
+- The `provider` property value can be only be `JohnDeere`
+:::
+
+### Resource
+
+Here's the properties available for a Provider Organization.
```
Comment (awimbush) [inline docs/provider_organizations.md:142]: EDIT: The properties available for a Provider Organization are:

File: docs/provider_organizations.md [hunk 4]
```diff
@@ -118,3 +131,236 @@ At Trimble it is not possible to differentiate which organizations are connected
...
+{
+  
+  "provider_org_id": "5206781", 
+  "provider_org_name": "TB Farms", 
+  "status": "SELECTED" 
+
+}
+```
+
+- The `provider_org_id` property is referecing the organization id value from the provider. It should be used while managing this resource.
```
Comment (awimbush) [inline docs/provider_organizations.md:154]: The 'provider_org_id' property references the organization ID value from the provider.  For example, if the John Deere organization ID is 12345, then the 'provider_org_id' should be 12345.

File: docs/provider_organizations.md [hunk 5]
```diff
@@ -118,3 +131,236 @@ At Trimble it is not possible to differentiate which organizations are connected
...
+- The `provider_org_name` property is referecing the organization name value from the provider.
```
Comment (awimbush) [inline docs/provider_organizations.md:155]: The 'provider_org_name' property references the value of the organization name from the provider.

File: docs/provider_organizations.md [hunk 6]
```diff
@@ -118,3 +131,236 @@ At Trimble it is not possible to differentiate which organizations are connected
...
+- The `status` property is refrencing to the current state of the resource. The property can have multiple values between `PREVIEW`, `SELECTED` and `BLOCKED`.
```
Comment (awimbush) [inline docs/provider_organizations.md:156]: The 'status' property represents the current state of the resource.  The state of the resource can be either 'PREVIEW', 'SELECTED', or 'BLOCKED'.

File: docs/provider_organizations.md [hunk 7]
```diff
@@ -118,3 +131,236 @@ At Trimble it is not possible to differentiate which organizations are connected
...
+  "provider_org_name": "TB Farms", 
+  "status": "SELECTED" 
+
+}
+```
+
+- The `provider_org_id` property is referecing the organization id value from the provider. It should be used while managing this resource.
+- The `provider_org_name` property is referecing the organization name value from the provider.
+- The `status` property is refrencing to the current state of the resource. The property can have multiple values between `PREVIEW`, `SELECTED` and `BLOCKED`. 
+  - If the `status` property has the value `SELECTED`, all the resources available to be fecthed from the provider will be processed.
```
Comment (awimbush) [inline docs/provider_organizations.md:157]: If the 'status' property has the value 'SELECTED', all available resources will be fetched from the provider and processed.

File: docs/provider_organizations.md [hunk 8]
```diff
@@ -118,3 +131,236 @@ At Trimble it is not possible to differentiate which organizations are connected
...
+  - If the `status` property has the value `PREVIEW`, no resources beyond the organization itsefl will be processed.
```
Comment (awimbush) [inline docs/provider_organizations.md:158]: @gustavorodriguespereira I don't understand what this means.

File: docs/provider_organizations.md [hunk 9]
```diff
@@ -118,3 +131,236 @@ At Trimble it is not possible to differentiate which organizations are connected
...
+  - If the `status` property has the value `BLOCKED`, the organization does not have the required permissions in order to be processed by Leaf API, to troubleshoot this issue please review the Step 3 of the [John Deere Authentication Guide][7], once the setup is complete, the resource will be available with the `PREVIEW` status. For any additional troubleshoot options, please reach out to help@withleaf.io
```
Comment (awimbush) [inline docs/provider_organizations.md:159]: - If the `status` property has the value `BLOCKED`, the organization does not have the required permissions to be processed by Leaf.  To troubleshoot this issue please review Step 3 of the [John Deere Authentication Guide][7], once the setup is complete, the resource will be available with the `PREVIEW` status. For any additional troubleshooting options, please reach out to help@withleaf.io

File: docs/provider_organizations.md [hunk 10]
```diff
@@ -118,3 +131,236 @@ At Trimble it is not possible to differentiate which organizations are connected
...
+    "provider_org_name": "Agriculture Organization",
+    "status": "PREVIEW"
+  }
+]
+
+```
+
+### Get a Provider Organization
+
+With this endpoint, users can select a provider organization individually.
```
Comment (awimbush) [inline docs/provider_organizations.md:237]: With this endpoint users can retrieve individual provider organizations.

File: docs/provider_organizations.md [hunk 11]
```diff
@@ -118,3 +131,236 @@ At Trimble it is not possible to differentiate which organizations are connected
...
+- The `provider_org_name` property is referecing the organization name value from the provider.
+- The `status` property is refrencing to the current state of the resource. The property can have multiple values between `PREVIEW`, `SELECTED` and `BLOCKED`. 
+  - If the `status` property has the value `SELECTED`, all the resources available to be fecthed from the provider will be processed.
+  - If the `status` property has the value `PREVIEW`, no resources beyond the organization itsefl will be processed.
+  - If the `status` property has the value `BLOCKED`, the organization does not have the required permissions in order to be processed by Leaf API, to troubleshoot this issue please review the Step 3 of the [John Deere Authentication Guide][7], once the setup is complete, the resource will be available with the `PREVIEW` status. For any additional troubleshoot options, please reach out to help@withleaf.io
+
+
+### Get All Provider Organizations
+
+With this endpoint users can check the organizations ingested from the integration process.
```
Comment (awimbush) [inline docs/provider_organizations.md:164]: With this endpoint users can retrieve all organizations.

**Review by awimbush — CHANGES_REQUESTED** [review]
Please make edits.

**Review by awimbush — APPROVED** [review]

**Pull request merged**